index.js

```diff
@@ -1 +1,95 @@
 // implement your API here
+
+
+const express = require('express');
+
+const db = require('./data/db');
+const port = 4000;
+const server = express();
+
+server.use(express.json());
+
+
+server.get("/api/users", (req, res) => {
+    const id = req.params.id;
+    db
+      .find()
+      .then(user => {
+        if (user) {
+          res.status(200).json({ success: true, user });
+        } else {
+          res
+            .status(404)
+            .json({ success: false, message: "could not find that id" });
+        }
+      })
+      .catch(({ code, message }) => {
+        res.status(code).json({ success: false, message });
```

Comment:
```suggestion
        res.status(500).json({ success: false, message });
```

I would like to see you use the actual status number for a while, at least until you have a lot of the status numbers memorized.